src/graph_compute/common.h

```diff
@@ -64,7 +90,7 @@
 #endif
 
 #ifndef NUMA_WORKERS
-  #define NUMA_WORKERS 1
+  #define NUMA_WORKERS 0
```

Comment:
NUMA_WORKERS and NUMA_INIT are used by the numa allocator, which can be used by all scheduling techniques.  The defaults set in lines 76-80 are for convenience with D1_NUMA if the user wants to not specify the specific number of workers and wants to merely specify PARALLEL, as with the other scheduling techniques.